index.html

```diff
@@ -63,7 +63,9 @@ <h2 class="text-dec">KamandPrompt</h2>
             </div>
           </div>
       </div>
-      <div class="small-text no-style bottom-center"><a href="#about">Scroll down to know more</a></div>
+      <div class="small-text no-style bottom-center">
+        <a class="scroll-down" href="#about">Scroll down</a>
```

Comment:
@viciouskitten Can you change the text to `Scroll down to know more` ?

Reply:
Would you accept "Scroll down to learn more?" i think it sounds better. Happy to change it back if you don't agree though :)

Reply:
Cool. That sounds fine.